lib/utils.js

```diff
@@ -11,6 +11,8 @@ const operatorsArray = _.values(operators);
 
 let inflection = require('inflection');
 
+exports.implicitNew = require('./utils/implicitNew');
```

Comment:
There is a method `classToInvokable` in these utils, which is same as `implicitNew`. Either use that old method or get rid of  `classToInvokable` so we only use one common method